### PR TITLE
Fix delayed DrawImage()

### PR DESCRIPTION
**Pull request by hiroebe**
`resolvePendingPixels` must be called in `delayedCommands`.
Otherwise, following `init` function does not work:

```go
func init() {
    srcImg, _ := ebiten.NewImage(w, h, ebiten.FilterDefault)
    dstImg, _ := ebiten.NewImage(w, h, ebiten.FilterDefault)

    // Set some pixels
    srcImg.Set(16, 16, clr)

    // Previous Set() is ignored, and nothing is drawn on dstImg.
    dstImg.DrawImage(srcImg, nil)
}
```

### CODE REVIEW

**Comment by hajimehoshi** [comment]
Thanks! Would it be possible to add tests at `image_test.go`?

**Comment by hiroebe** [comment]
OK, I will.

**Comment by hajimehoshi** [comment]
I realized that the test should be done before the main loop, then this might be tricky. I'll do if you are not sure.

**Comment by hiroebe** [comment]
I think I understand the trickiness, but adding tests to `image_test.go` caused panic in glfw and I'm struggling with it. So could you please add test instead of me?

File: internal/buffered/image.go [hunk 1]
```diff
@@ -222,6 +222,8 @@ func (i *Image) DrawImage(src *Image, bounds image.Rectangle, a, b, c, d, tx, ty
 	delayedCommandsM.Lock()
 	if needsToDelayCommands {
 		delayedCommands = append(delayedCommands, func() {
+			src.resolvePendingPixels(true)
+			i.resolvePendingPixels(false)
```
Comment (hajimehoshi) [inline internal/buffered/image.go]: Could you add the same thing in `DrawTriangles`?

Comment (hiroebe) [reply internal/buffered/image.go]: Done.

**Pull request merged**